dev/PackageManager/API/M.W.M.D.PackageDeploymentManager.cpp

```diff
@@ -54,14 +54,15 @@ namespace winrt::Microsoft::Windows::Management::Deployment::implementation
         {
             if (!IsReady(packageSetItem))
             {
-                Common::Logging::DebugLog(std::format(L"Id={} PackageFamilyName={} MinVersion={}.{}.{}.{} ArchitectureFilter:0x{}",
-                            packageSetItem.Id().c_str(),
-                            packageSetItem.PackageFamilyName().c_str(),
-                            packageSetItem.MinVersion().Major,
-                            packageSetItem.MinVersion().Minor,
-                            packageSetItem.MinVersion().Build,
-                            packageSetItem.MinVersion().Revision,
-                            static_cast<std::uint32_t>(packageSetItem.ProcessorArchitectureFilter())));
+                (void)LOG_HR_MSG(MSIXPACKAGEMANAGER_E_PACKAGE_SCAN_FAILED,
```

Comment:
LOG_HR_MSG is what Pratik was trying to avoid, as it clutters OutputDebugString logging with meaningless HRESULT descriptions.

Reply:
how was this resolved?